infra/notifications/src/models/identity.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
```

Comment:
To not duplicate stuff I think it would be better to import this identity: https://github.com/OriginProtocol/origin/blob/master/infra/identity/src/models/identity.js

This way if the database model changes, we are less likely to cause errors since we only have to change it in 1 place

Reply:
Agreed. @franckc and I talked about re-working how we handle models that are used across packages/apps. I lean towards having a central place where all our db models and migrations are stored. Right now its such a patchwork, making migrations hard, plus making it difficult to know what models are used in which packages.apps.

Reply:
Oh, and yes @sparrowDom , changed code to use the one Identity model in https://github.com/OriginProtocol/origin/commit/623fa7431d33a0fe02bbe597e4fff1bcb1858096